lib/Installation/Popups/InstallBackPopup.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces methods to handle the
+# installation popup. ([install] [Back])
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::Popups::InstallBackPopup;
```

Comment:
this is an OKCancelPopup, which could be generic and reused somewhere else, just the labels of the button might differ.